editor/js/Config.js

```diff
@@ -14,7 +14,7 @@ function Config() {
 		'project/renderer/antialias': true,
 		'project/renderer/shadows': true,
 		'project/renderer/shadowType': 1, // PCF
-		'project/renderer/physicallyCorrectLights': false,
+		'project/renderer/useLegacyLights': false,
```

Comment:
I don't think we should change the default yet?

Reply:
That depends on @mrdoob preference. It could be interesting to enable physically correct lights in the editor earlier compared to the library itself.